src/app.js

```diff
@@ -90,7 +90,11 @@ const imgSrc = [
   'https://*.googletagmanager.com',
   'https://firefoxusercontent.com',
   'https://mozillausercontent.com/',
-  'https://monitor.cdn.mozilla.net/'
+  'https://monitor.cdn.mozilla.net/',
+  // Support OptinMonster Campaign Embed Code:
+  // https://optinmonster.com/docs/how-to-manually-add-an-after-post-or-inline-optin/#Campaign_Embed_Code
+  ' data:',
```

Comment:
Hm, unfortunate that we need this but I don't think there's a way to restrict `data:` further unfortunately.

Reply:
Oh I'd missed this one, but this feels like a fairly significant loosening of restrictons, opening us up to embedding of images that abuse vulnerabilities in image formats, or e.g. SVGs that include JavaScript?

Not sure how big of a risk that is, assuming that we trust all scripts that we do allow to run on this page :shrug: 

(I think theoretically